packages/models/src/bitcoin-unit.model.ts

```diff
@@ -0,0 +1,26 @@
+export enum WalletBitcoinUnit {
+  bitcoin = 'bitcoin',
+  satoshi = 'satoshi',
+}
+export type BitcoinUnit = keyof typeof WalletBitcoinUnit;
+
+export type BitcoinUnitSymbol = 'BTC' | 'sat';
+
+export interface BitcoinUnitInfo {
+  name: WalletBitcoinUnit;
+  symbol: BitcoinUnitSymbol;
+  decimal: string;
+}
+
+export const bitcoinUnitsKeyedByName: Record<WalletBitcoinUnit, BitcoinUnitInfo> = {
+  [WalletBitcoinUnit.bitcoin]: {
+    name: WalletBitcoinUnit.bitcoin,
+    symbol: 'BTC',
+    decimal: '1',
+  },
+  [WalletBitcoinUnit.satoshi]: {
+    name: WalletBitcoinUnit.satoshi,
+    symbol: 'sat',
+    decimal: '0.00000001',
```

Comment:
Went with a string here, we will have to use `initBigNumber` helper when used, thoughts?

Reply:
I think thats OK